core/src/main/resources/ssl/self_signed_fips/README.txt

```diff
@@ -0,0 +1,34 @@
+//generate a NSS keystore (note that key is generated on the fly)
```

Comment:
This file looks to me more than a script than a README. After reading it I have no idea what's going on and what these files are for.

Reply:
The README is the counterpart of the existing https://github.com/web-servers/noe-core/blob/master/core/src/main/resources/ssl/self_signed/README.txt with the relevant information to recreate the certificates in the FIPS case. Like the existing one was.

Reply:
Done